examples/navier-stokes/advection.h

```diff
@@ -210,7 +210,7 @@ static int Advection(void *ctx, CeedInt Q,
       dv[i+(4+0*5)*Q] = 0;
       dv[i+(4+1*5)*Q] = 0;
       dv[i+(4+2*5)*Q] = 0;
-      v[i+4*Q]   = E*(du[0]*wBJ[0] + du[3]*wBJ[1] + du[6]*wBJ[2]);
+      v[i+4*Q]   =-E*(du[0]*wBJ[0] + du[3]*wBJ[1] + du[6]*wBJ[2]);
```

Comment:
I agree that this is needed. We might want to avoid using `-=` in the future because it breaks symmetry among terms. I'm not sure why not to create one multiline expression in this case.

Reply:
I have no idea why we did it this way in the past, but I can change that easily.

Reply:
Updated